cmd/server.go

```diff
@@ -40,20 +41,47 @@ var serverCmd = &cobra.Command{
 					},
 				}
 				serverinfos = append(serverinfos, fill...)
-
 			}
 			if idFlag {
 				rowsToDisplay = len(heading)
 			}
 			render.Table(out, heading[:rowsToDisplay], serverinfos)
-
 		} else {
 			render.AsJSON(out, servers)
 		}
 		fmt.Print(out)
 	},
 }
 
+var onCmd, offCmd = &cobra.Command{
+	Use:   "on",
+	Short: "Turn server on using id",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := context.Background()
+		client.StartServer(ctx, args[0])
+	},
+}, &cobra.Command{
+	Use:   "off",
+	Short: "Turn server off using id",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := context.Background()
+		if forceFlag {
+			err := client.StopServer(ctx, args[0])
+			if err != nil {
+				panic(err)
+			}
+			return
+		}
+		err := client.ShutdownServer(ctx, args[0])
+		if err != nil {
+			panic(err)
+		}
+		return
+	},
+}
+
 func init() {
+	serverCmd.AddCommand(onCmd, offCmd)
+	rootCmd.PersistentFlags().BoolVarP(&forceFlag, "force", "f", false, "Force shutdown server using id")
```

Comment:
so, you need to `serverCmd.Flags().String("uuid", "", "uuid of a server")` in the init function of the serverCmd

Reply:
replace `uuid` with `id` in case of this applies https://github.com/gridscale/gscloud/issues/35